src/instructlab/model/serve_backend.py

```diff
@@ -80,11 +114,25 @@ def serve_backend(
     elif backend == backends.VLLM:
         # First Party
         from instructlab.cli.model.serve import warn_for_unsupported_backend_param
-        from instructlab.utils import contains_argument
 
         warn_for_unsupported_backend_param(ctx)
 
         ctx.obj.config.serve.vllm.vllm_args = ctx.obj.config.serve.vllm.vllm_args or []
+
+        # Fix ValueError: Too large swap space. 16.00 GiB out of the 15.01 GiB total CPU memory is allocated for the swap space.
+        check_size, effective_gpus, available_gpus = check_tensor_parallel_size(
```

Comment:
check_size is a confusing name to me.  Seems to indicate something like: tps_size_invalid